presto-main/src/main/java/com/facebook/presto/server/ServerMainModule.java

```diff
@@ -554,6 +555,9 @@ public ListeningExecutorService createResourceManagerExecutor(ResourceManagerCon
         jsonBinder(binder).addKeySerializerBinding(VariableReferenceExpression.class).to(VariableReferenceExpressionSerializer.class);
         jsonBinder(binder).addKeyDeserializerBinding(VariableReferenceExpression.class).to(VariableReferenceExpressionDeserializer.class);
 
+        // history statistics
+        binder.bind(HistoryBasedPlanStatisticsManager.class).in(Scopes.SINGLETON);
```

Comment:
should this depend on configuration? or does it not matter since it will fall back to the composable stats calculator?

Reply:
Falling back is already controlled via session and configs using ifs